src/admin/components/website/modalParts/emailOverride/WebsiteOverrideSettings.tsx

```diff
@@ -17,58 +17,61 @@
 //
 
 import * as React from 'react';
+import Validator from 'admin/services/Validation';
+import * as Immutable from 'seamless-immutable';
 import {LabeledInput} from 'admin/components/website/additionalComponents/LabeledInput';
 import {SwitchBox} from 'admin/components/website/additionalComponents/SwitchBox';
+import {TagList} from 'admin/components/website/additionalComponents/TagList';
 
-export interface IWebsiteOverrideSettingsContentSection {
+export interface IWebsiteOverrideSettings {
 	list: string[];
 	listPropName: string;
-	onSubmit: (data: any) => void;
-	checkValidation: (value: string, touched: boolean) => string;
+	isControllable?: boolean;
 	controlPropName?: string;
 	controlPropValue?: boolean;
+	label: JSX.Element | string;
+	ID: string;
+	value: string;
+	touched: boolean;
+	onSubmit: (data: any) => void;
+	onChange: (value:string, touched:boolean) => void;
```

Comment:
is it used in this component? may be I missed

Reply:
95 line
https://github.com/dbmedialab/reader-critics/pull/771/files/274414e92804c7ba742c3eb0621c8bb1ac7ffa4e#diff-6f649265f4a0439ed794954bdc7f8b3fR92